src/utils/map.go

```diff
@@ -1,12 +1,25 @@
 package utils
 
-import "github.com/iancoleman/orderedmap"
+import (
+	"github.com/iancoleman/orderedmap"
+	"keboola-as-code/src/json"
+)
 
 type Pair struct {
 	Key   string
 	Value interface{}
 }
 
+func ConvertByJson(input, target interface{}) {
```

Comment:
Helper, .... zakoduje `input` do JSON, a dekoduje do `target` ... tak mozem previest nejaku `struct` na `map` a podobne.
Priamo v `go` pre to podpora nie je ... su nejake kniznice co to robia nie cez JSON, ale cez `reflection`, ale nefunguju velmi dobre.

Pouzivam to, ked chcem ulozit `[]*Blocks` do JSON konfiguracie, kde to potrebujem najprv previest na `map`.